schemas/specimen.json

```diff
@@ -349,9 +351,9 @@
       },
       "restrictions": {
         "codeList": [
-           "Genomics",
-           "Image analysis",
-           "Pathology estimate by percent nuclei"
+          "Genomics",
+          "Image analysis",
+          "Pathology estimate by percent nuclei"
```

Comment:
Came looking for these values, for the tests... ended up cleaning some whitespace 👍